src/functionalTest/resources/scenarios/RIA-6557-ada-notification-submitted-hearing-requirements.json

```diff
@@ -0,0 +1,65 @@
+{
+  "description": "RIA-1283 Notification to case officer and legal rep of submitted hearing requirements",
```

Comment:
Can you change the description to RIA-6557 (add ADA as well), same as your file name.
This will get printed when we run functional tests

Reply:
ah yes, updated in latest commit.